kytos/core/interface.py

```diff
@@ -269,6 +278,40 @@ def remove_tag_ranges(self, tag_type: str):
             )
             self.tag_ranges[tag_type] = self.default_tag_values[tag_type]
 
+    def set_special_tagss(
```

Comment:
@Alopalao, looks like it was a typo with extra `s` on this method if you could also unit test this, then it can get merged.

Reply:
Fixed.

Reply:
Added test